src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -760,9 +767,48 @@ private protected override void SaveModel(ModelSaveContext ctx)
             for (int i = 0; i < _ldas.Length; i++)
             {
                 _ldas[i].Save(ctx);
+
+                if(_columns[i].OutputTopicWordSummary)
+                    SaveTopicWordSummary(ctx, i);
             }
         }
 
+        private void SaveTopicWordSummary(ModelSaveContext ctx, int i)
+        {
+            var summary = GetLdaDetails(i);
+
+            ctx.SaveTextStream(WordTopicModelFilename, writer =>
+            {
+                if (summary.WordScoresPerTopic != null)
+                {
+                    int topId = 0;
+                    foreach (var wordScores in summary.WordScoresPerTopic)
+                    {
+                        foreach (var wordScore in wordScores)
+                        {
+                            writer.WriteLine($"Top[{topId}]: {wordScore.Word}\t{wordScore.Score}");
```

Comment:
nit: I think it might be clearer to use `$"Topic[{topId}]` since "Top" might be somewhat misguiding.